### DISCUSSION TITLE
Is there any way to have an image on the front of the card?

### DISCUSSION THREAD

**Discussion by cjappl** [2anki/server#1454] [Q&A] [answered]
Trying to design some cards with images on the front, but when I try to past an image into the > section of a notion dropdown, it allows me to put an image above the card, or in the "answer" of the card.

Is there something I'm missing?

**Comment by aalemayhu** [comment, marked as answer]
You can use columns to bring images to the front https://youtu.be/4zXcupiUhy4?si=-hfOZm67Zt1qklHH

**Reply by cjappl** [reply]
Thank you!! That's great.

Is there a way to do that without the integration? (just the HTML import). Is the Notion integration "the future" and I'm missing out on features by not using it?

Thanks for the excellent tool.

**Reply by aalemayhu** [reply]
No unfortunately not. I am not sure how we can get it reliably from the HTML but I will create a GitHub issue you can follow https://github.com/2anki/2anki.net/issues/1478

**Reply by cjappl** [reply]
Great, thank you!
